tests/src/core/Transform.test.ts

```
@@ -59,4 +60,85 @@ describe("Transform test", function () {
+    expect(entity1.transform.scale).to.deep.include({ x: 4, y: 5, z: 6 });
+  });
 });
+
+class SubClassOfTransform extends Transform {
+  size: Vector2 = new Vector2();
+
+  /**
+   * @internal
+   */
+  _generateTransitionState(state: any): void {
+    state.position ? state.position.copyFrom(this.position) : (state.position = this.position.clone());
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)